src/examples/table/TableVerticalScrolling.tsx

```diff
@@ -18,15 +18,17 @@ const rowData = Array.from(Array(10)).map((row, index) => ({
   soil: 'Well draining'
 }));
 
-const StyledHeaderCell = styled(HeaderCell)`
-  box-shadow: inset 0 -1px 0 ${props => getColorV8('neutralHue', 300, props.theme)};
+const StyledContainer = styled.div`
+  max-height: 400px;
+  overflow-x: auto;
+  color-scheme: only ${p => p.theme.colors.base};
```

Comment:
All of these `color-scheme` CSS properties is making it apparent that the `<Table>` component itself should have this CSS and should be removed from all of these examples. Doing so will apply nicely to all of the product usage of Garden scrolling tables.

Reply:
This is a challenging scenario because Garden `Table` has no scroll by default (same as any other basic HTML table, also the reason W3C's [only article](https://www.w3schools.com/howto/howto_css_table_responsive.asp) suggests using a container, same as we do here).

We would need to provide some additional API to apply `overflow` CSS to `Table`/`Table.Body` ([one](https://www.geeksforgeeks.org/how-to-create-table-with-100-width-with-vertical-scroll-inside-table-body-in-html/), [two](https://vembarrajan.medium.com/html-css-tricks-scroll-able-table-body-tbody-d23182ae0fbc)).

So, all that said, perhaps we can do both: provide scrolling via `Table` prop + move `color-scheme` into `CodeExample.tsx` (which would apply to all website examples).

Reply:
Oh, you're totally right @geotrev – there's not a quality addition to the `Table` component 🤦